Makefile

```diff
@@ -187,7 +187,11 @@ live: $(LIVE_IMG).img
 installer: $(INSTALLER_IMG).raw
 installer-iso: $(INSTALLER_IMG).iso
 
-$(CONFIG_IMG): conf/server conf/onboard.cert.pem conf/wpa_supplicant.conf conf/authorized_keys conf/ | $(DIST)
+CONFIG_FILES = conf/server conf/onboard.cert.pem conf/wpa_supplicant.conf \
+	       conf/authorized_keys conf/UsbInvocationScript.txt \
+	       conf/
+
+$(CONFIG_IMG):  $(CONFIG_FILES) | $(DIST)
```

Comment:
Intersting @zach-welch-timesys -- so it appears that putting a magic UsbInvocationScript.txt into the CONFIG partition triggers Dell BIOS to do the right thing. I'm actually not a big fan of it being there though -- since it'll be permanently placed in the final installed image CONFIG partition as well and we should avoid that.

Could you please check whether we can use the UEFI partition on the installer media to achieve the same effect?

Reply:
I have moved this file to the EFI partition, and it continues to function as intended.

Reply:
@zach-welch-timesys -- great! In that case -- I suggest you re-work this PR to add this UsbInvocationScript.txt to pkg/mkimage-raw-efi -- since that is the right place to accumulate knowledge of various boards/units on which we install

Reply:
Done